fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/Loan.java

```diff
@@ -2668,8 +2668,11 @@ private void compareDisbursedToApprovedOrProposedPrincipal(BigDecimal disbursedA
 
         if (this.loanProduct().isDisallowExpectedDisbursements() && this.loanProduct().isAllowApprovedDisbursedAmountsOverApplied()) {
             BigDecimal maxDisbursedAmount = getOverAppliedMax();
-            if (disbursedAmount.compareTo(maxDisbursedAmount) > 0) {
-                final String errorMessage = "Loan disbursal amount can't be greater than maximum applied loan amount calculation.";
+            if (totalDisbursed.compareTo(maxDisbursedAmount) > 0) {
```

Comment:
Are you sure the `totalDisbursed` is containing this "newly" disbursed amount already?

Reply:
Yes, I just double checked it, and it does contain the newly disbursed amount as well.